lib/widgets/content.dart

```diff
@@ -222,17 +251,20 @@ const double kBaseFontSize = 17;
 /// This does not include metadata like the sender's name and avatar, the time,
 /// or the message's status as starred or edited.
 class MessageContent extends StatelessWidget {
-  const MessageContent({super.key, required this.message, required this.content});
+  const MessageContent({super.key, required this.message, required ZulipMessageContent content}) : _content = content;
```

Comment:
This is for type promotion, similar to [this other instance](https://github.com/zulip/zulip-flutter/pull/808#issue-2406389310).

Reply:
> narrow is refactored to become private because otherwise `switch` is not able to promote the type of it.

I see. In that case:
https://github.com/zulip/zulip-flutter/pull/808#discussion_r1677245004

:slightly_smiling_face: 

In general, there's a local fix for that problem at any place where it arises, whereas changing a field from public to private changes the class's API for everywhere. So it's better to not let this affect whether the field is public or private.